### ISSUE TITLE
chectl workspace:start fails for multi-user Che

### ISSUE THREAD

**Issue by l0rd** [eclipse-che/che#13828]
### Description

The command `workspace:start` doesn't  have a parameter `--access-token=access-token` (`server:stop` already accepts this parameter). 

Hence when trying to start a workspace using chectl on multi-user Che we get an error `E_CHE_API_UNAUTHORIZED`

### Reproduction Steps

```
$ chectl server:start --installer operator -m -p minishift
(...)
$ chectl workspace:start -f <(chectl devfile:generate --name=minimal-wksp)
(...)
Error: E_CHE_API_UNAUTHORIZED - Endpoint: http://che-che.192.168.64.71.nip.io/api/workspace/devfile - Message: "Authorization token is missed"
```

**Comment by l0rd** [comment]
This is a step of the happy path test so I consider that it should be fixed for che 7.0.0 cc @rhopp

**Comment by rhopp** [comment]
So far we are debugging/fine tuning happy path on singleuser instance, but yeah... this should be added.
